src/components/Category/AllergenFriendly.js

```diff
@@ -0,0 +1,51 @@
+
+import React from 'react';
+import Type from '../Type/Type';
+import './Category.css';
+
+const AllergenFriendly = ({ allergenFriendly }) => {
+    const filterMakeupTypes = allergenFriendly.map((item) => item["product_type"]);
+    const productTypes = filterMakeupTypes.filter(
+      (item, index) => filterMakeupTypes.indexOf(item) === index
+    );
+
+    const assignUrl = (item) => {
```

Comment:
Thank for you driving during this time Connie! I'm glad it worked out and we were able to apply standard styling to everything so far.